docs/how_to/render_plane_manipulator.md

```diff
@@ -19,7 +19,7 @@ the rendering plane.
      napari
      ```
 
-2. Load the "HIV virus-like particles tomogram" image from the File menu: File -> Open Sample -> HIV virus-like particle tomogram. Note that this will download the sample image so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
+2. Load the "HIV virus-like particles tomogram" image from the File menu: **File -> Open Sample -> HIV virus-like particles tomogram**. Note that this will download the [sample image from Zenodo](https://doi.org/10.5281/zenodo.6504891) (~470 Mb) so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
```

Comment:
This part of the documentation is fragile, because as soon as there is more than one example dataset provided, then they'll be found in a subfolder, eg: File -> Open Sample -> napari-threedee -> HIV virus-like particle tomograms
I don't know what my suggestion is here, either we ignore this possibility for now, or force the change to happen sooner by adding a second sample dataset.

Reply:
Ah good point. I think we can leave as is and then update if/when we add additional sample data plugins